<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SUmusic</class>
 <widget class="QMainWindow" name="SUmusic">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>11</pointsize>
    <bold>true</bold>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>SU云音乐</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icon.qrc">
    <normaloff>:/icon/4SUmusic/rainy_headicon.png</normaloff>:/icon/4SUmusic/rainy_headicon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
	border:none;
}

QPushButton:hover
{
	background-color:rgb(212,212,212);
}


QMainWindow{         /*类名*/
    background-color:rgb(243,244,246); 
    border-radius: 10px;             /*四角圆角的直径*/
    border-style:solid;             /*边框样式（实线）*/
    border-width:2px;             /*边框宽度*/
    border-color:rgb(213, 213, 213);
}

QPushButton#closeWindow:hover{
	background-color:red;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="Musiclist">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>281</width>
      <height>441</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>560</y>
      <width>611</width>
      <height>80</height>
     </rect>
    </property>
    <widget class="QPushButton" name="music_start">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Icon.qrc">
       <normaloff>:/icon/4SUmusic/codicon_debug-start.png</normaloff>
       <normalon>:/icon/4SUmusic/ic_baseline-pause.png</normalon>:/icon/4SUmusic/codicon_debug-start.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="music_previews">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/icon/4SUmusic/mi_previous.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="music_next">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/icon/4SUmusic/mi_next.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="all_long">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>470</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="reload">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>刷新列表</string>
    </property>
    <property name="icon">
     <iconset resource="Icon.qrc">
      <normaloff>:/icon/4SUmusic/material-symbols_refresh-rounded.png</normaloff>:/icon/4SUmusic/material-symbols_refresh-rounded.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="now_long">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>470</y>
      <width>76</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="yinliang">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>620</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>音量</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="musicslider">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>510</y>
      <width>791</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #e9dff0; /* 更亮的浅紫边框 */
    background: #f8f6fc; /* 更亮的浅灰紫色槽道 */
    height: 8px; /* 槽道高度 */
    border-radius: 4px; /* 圆角槽道 */
}

QSlider::sub-page:horizontal {
    background: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:1, y2:0,
        stop:0 #f0c8dd, /* 更亮的浅粉紫 */
        stop:1 #e6b2d1  /* 更亮的紫粉色 */
    ); /* 已播放部分渐变 */
    border: 1px solid #e6b2d1; /* 较亮的渐变边框 */
    height: 8px;
    border-radius: 4px;
}

QSlider::add-page:horizontal {
    background: #fefcff; /* 更亮的未播放部分浅灰粉色 */
    border: 1px solid #e9dff0; /* 更亮的浅紫边框 */
    height: 8px;
    border-radius: 4px;
}

QSlider::handle:horizontal {
    background: qradialgradient(
        cx:0.5, cy:0.5, radius:0.5,
        fx:0.5, fy:0.5,
        stop:0 #fff, /* 白色中心 */
        stop:1 #f6c4d3 /* 更亮的浅粉外圈 */
    ); /* 滑块渐变颜色 */
    border: 1px solid #f0a3b8; /* 更亮的粉色边框 */
    width: 14px; /* 滑块宽度 */
    height: 14px; /* 滑块高度 */
    margin: -3px 0; /* 调整垂直位置 */
    border-radius: 7px; /* 圆形滑块 */
}

QSlider::handle:horizontal:hover {
    background: qradialgradient(
        cx:0.5, cy:0.5, radius:0.5,
        fx:0.5, fy:0.5,
        stop:0 #fff8fa, /* 悬浮时更亮的中心 */
        stop:1 #f89ab6  /* 更亮的外圈亮粉色 */
    ); /* 鼠标悬浮滑块渐变 */
    border: 1px solid #f48ca2; /* 边框更亮 */
}

QSlider::groove:horizontal:hover {
    background: #f9f3f9; /* 鼠标悬浮时更亮的浅紫色 */
}
</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="about">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
   <widget class="QLabel" name="MuCover">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>120</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Icon.qrc">:/icon/4SUmusic/default_cover.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="usrInput">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>711</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="undoRedoEnabled">
     <bool>true</bool>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::LineWrapMode::WidgetWidth</enum>
    </property>
    <property name="overwriteMode">
     <bool>false</bool>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="addMusic">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>40</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>添加音乐</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::EditFind"/>
    </property>
   </widget>
   <widget class="QPushButton" name="open_file">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>打开文件夹</string>
    </property>
    <property name="icon">
     <iconset resource="Icon.qrc">
      <normaloff>:/icon/4SUmusic/mdi-light_file.png</normaloff>:/icon/4SUmusic/mdi-light_file.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="del_music">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除音乐</string>
    </property>
    <property name="icon">
     <iconset resource="Icon.qrc">
      <normaloff>:/icon/4SUmusic/material-symbols_delete-outline-rounded.png</normaloff>:/icon/4SUmusic/material-symbols_delete-outline-rounded.png</iconset>
    </property>
   </widget>
   <widget class="QSlider" name="volum_slide">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>550</y>
      <width>20</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical {
    border: 1px solid #e9dff0; /* 边框浅紫色 */
    background: #f8f6fc; /* 浅灰紫色槽道 */
    width: 8px; /* 槽道宽度 */
    border-radius: 4px; /* 圆角槽道 */
}

QSlider::sub-page:vertical {
    background: #fefcff; /* 已调整部分浅灰粉色 */
    border: 1px solid #e9dff0; /* 边框浅紫色 */
    width: 8px;
    border-radius: 4px;
}

QSlider::add-page:vertical {
    background: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #f0c8dd, /* 更亮的浅粉紫 */
        stop:1 #e6b2d1  /* 更亮的紫粉色 */
    ); /* 未调整部分渐变 */
    border: 1px solid #e6b2d1; /* 渐变边框 */
    width: 8px;
    border-radius: 4px;
}

QSlider::handle:vertical {
    background: qradialgradient(
        cx:0.5, cy:0.5, radius:0.5,
        fx:0.5, fy:0.5,
        stop:0 #fff, /* 白色中心 */
        stop:1 #f6c4d3 /* 更亮的浅粉外圈 */
    ); /* 滑块渐变颜色 */
    border: 1px solid #f0a3b8; /* 边框浅粉色 */
    width: 14px; /* 滑块宽度 */
    height: 14px; /* 滑块高度 */
    margin: 0 -3px; /* 调整水平位置 */
    border-radius: 7px; /* 圆形滑块 */
}

QSlider::handle:vertical:hover {
    background: qradialgradient(
        cx:0.5, cy:0.5, radius:0.5,
        fx:0.5, fy:0.5,
        stop:0 #fff8fa, /* 悬浮时更亮的中心 */
        stop:1 #f89ab6  /* 更亮的外圈亮粉色 */
    ); /* 鼠标悬浮滑块渐变 */
    border: 1px solid #f48ca2; /* 边框更亮 */
}

QSlider::groove:vertical:hover {
    background: #f9f3f9; /* 鼠标悬浮时更亮的浅紫色 */
}
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="headBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,204,229);
border-radius: 10px;             /*四角圆角的直径*/
border-width:2px;             /*边框宽度*/
border-color:rgb(213, 213, 213);
border-style:solid</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="closeWindow">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
    </property>
   </widget>
   <widget class="QPushButton" name="minWindow">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ListRemove"/>
    </property>
   </widget>
   <widget class="QLabel" name="Introduction">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SU云音乐</string>
    </property>
   </widget>
   <widget class="QComboBox" name="songMode">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>560</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>占用清除</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Icon.qrc"/>
 </resources>
 <connections/>
</ui>
